packages/addon-knobs/example/typescript/index.tsx

```diff
@@ -25,7 +25,9 @@ stories.add('with all knobs', () => {
   const dob = date('DOB', new Date('January 20 1887'));
 
   const bold = boolean('Bold', false);
-  const color = color('Color', 'black');
+  const selectedColor = color('Color', 'black');
```

Comment:
This is changed because TS should complain about block scoping when trying to override `color`. Updated to use a different name (which lines up with the storybook example).